spec/Overview.html

```diff
@@ -2062,9 +2061,6 @@ <h4>The deriveBits method</h4>
               <li>
                 <p>
                   Let |result| be a new {{ArrayBuffer}}
```

Comment:
Yeah, I opted against it as it made the resulting text a bit clunky. And, the [fetch](https://fetch.spec.whatwg.org/#dom-body-arraybuffer) and [XHR](https://xhr.spec.whatwg.org/#dom-xmlhttprequest-response) specs do the same, for example.

Reply:
Yes, afaict “a new ArrayBuffer” isn’t actually defined at this level, right? Something needs to hit [AllocateArrayBuffer](https://tc39.es/ecma262/multipage/structured-data.html#sec-allocatearraybuffer) somewhere to actually create a new ArrayBuffer object in the sense that `new ArrayBuffer()` in ES code would, and Web IDL’s [create an ArrayBuffer](https://webidl.spec.whatwg.org/#arraybuffer-create) alg takes care of that.

Reply:
OK, fair enough. I've updated the text :+1:

Reply:
I didn’t see the first reply about fetch & XHR here till now. The XHR one is definitely wrong because it links “new” to [create a new object implementing the interface](https://webidl.spec.whatwg.org/#new), and ArrayBuffer is not an interface. The fetch one doesn’t link it to anything and seems like it’s also a spec hiccup (though again, I’m not an editor/expert so YMMV w/ my analysis).

Reply:
I think for both of those specifications the text predates the IDL algorithm. Please file issues/PRs?

Reply:
@annevk issues opened for [Fetch](https://github.com/whatwg/fetch/issues/1675) and [XHR](https://github.com/whatwg/xhr/issues/376).